### PR TITLE
MINOR: improve MinTimestampTrackerTest and fix NPE when null element removed

### PR DESCRIPTION
**Pull request by dguy**

### CODE REVIEW

**Comment by dguy** [comment]
@guozhangwang @enothereska @mjsax

File: streams/src/test/java/org/apache/kafka/streams/processor/internals/MinTimestampTrackerTest.java [hunk 1]
```diff
@@ -17,77 +17,63 @@
...
+import static org.hamcrest.MatcherAssert.assertThat;
 
 import org.junit.Test;
 
 public class MinTimestampTrackerTest {
 
-    private Stamped<String> elem(long timestamp) {
-        return new Stamped<>("", timestamp);
-    }
+    private MinTimestampTracker<String> tracker = new MinTimestampTracker<>();
```
Comment (mjsax) [inline streams/src/test/java/org/apache/kafka/streams/processor/internals/MinTimestampTrackerTest.java:27]: nit: add `final`

File: streams/src/test/java/org/apache/kafka/streams/processor/internals/MinTimestampTrackerTest.java [hunk 2]
```diff
@@ -17,77 +17,63 @@
...
+        tracker.removeElement(null);
+    }
 
-        assertEquals(insertionIndex, removalIndex);
+    @Test(expected = NullPointerException.class)
+    public void shouldThrowNullPointerExceptionWhenTryingToAddNullElement() throws Exception {
+        tracker.addElement(null);
     }
 
+    private Stamped<String> elem(long timestamp) {
```
Comment (mjsax) [inline streams/src/test/java/org/apache/kafka/streams/processor/internals/MinTimestampTrackerTest.java]: nit: add `final`

**Review by mjsax — APPROVED** [review]
LGTM

**Review by guozhangwang — APPROVED** [review]
Merged to trunk.

**Pull request closed without merging**